package/src/components/ProgressiveImage/v1/ProgressiveImage.js

```diff
@@ -157,23 +157,22 @@ class ProgressiveImage extends Component {
     buffer.onload = () => {
       this._mounted && this.setState({ ready: true });
     };
-    buffer.src = src || (srcs && srcs.small);
+    buffer.src = src || (srcs && srcs.medium);
   }
 
   /**
    *
    * @method renderPicture
-   * @summary Renders a `picture` element with the provided theme breakpoints and `props.srcs`
+   * @summary Renders a `picture` element with the theme breakpoints and `props.srcs`.
    * @return {Element} - `picture`
    */
   renderPicture() {
     const { altText, srcs } = this.props;
     return (
       <picture>
-        <source media={`(${getFromTheme(this.props, "rui_bp_md")})`} srcSet={srcs && srcs.large} />
-        <source media={`(${getFromTheme(this.props, "rui_bp_sm")})`} srcSet={srcs && srcs.medium} />
+        <source media={`(${getFromTheme(this.props, "rui_bp_sm")})`} srcSet={srcs && srcs.large} />
```

Comment:
@aldeed I changed the images that get displayed with ProgressiveImage, so that it shows the medium size image until 600px, then the large image. Previously, I was seeing somewhat pixelated images on mobile and desktop.